Makefile

```diff
@@ -37,6 +37,15 @@ FFI_DEPS:=$(addprefix $(FFI_PATH),$(FFI_DEPS))
 
 $(FFI_DEPS): build/.filecoin-install ;
 
+ifeq ($(shell uname -s), Darwin)
+export CGO_ENABLED=1
+    ifeq ($(shell uname -m), arm64)
+export GOARCH=arm64
+export LIBRARY_PATH="$(brew --prefix)/lib"
+export FFI_BUILD_FROM_SOURCE=1
+    endif
+endif
```

Comment:
In the Mac M1 docs we also export PATH as follows:

    export PATH="$(brew --prefix coreutils)/libexec/gnubin:/usr/local/bin:$PATH"

Is that not missing here?

Reply:
Oh thanks for the catch. I've updated it to include the `PATH` export now with `:=` to make sure that it's evaluated / immediately set.